membership-attribute-service/test/controllers/AttributeControllerTest.scala

```diff
@@ -242,7 +265,7 @@ class AttributeControllerTest extends Specification with AfterAll with Idiomatic
                  """.stripMargin)
   }
 
-  private def verifySuccessfullOneOfContributionsResult(result: Future[Result]) = {
+  private def verifySuccessfulOneOfContributionsResult(result: Future[Result]) = {
```

Comment:
If you're gonna repair the two 'l's at the end of Successful you may as well also add the additional 'f' at the end of 'OneOf' or call it 'OneTime'!